### PR TITLE
Add default PR template

### PR DESCRIPTION
**Pull request by ben-moss**
Test of branch protection and intro of new PR template

### CODE REVIEW

**Review by peterchoo — APPROVED** [review]
:+1:

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,3 +1,40 @@
...
+
+## Test Instructions
+
+<!-- List the instructions the reviewer will need to follow to ensure this works correctly. -->
+<!-- Try to make it easy for the reviewer to test this PR. -->
+
+## Acceptance Criteria
+
+<!-- How can the reviewer know the aims of the PR have been achieved? -->
+<!-- Ensure any post-test actions needed for confirmation are included here. -->
```
Comment (robfallows) [inline .github/PULL_REQUEST_TEMPLATE.md:34]: I'm increasingly of the opinion these should be merged into

    ## Test Instructions / Acceptance Criteria

**Review by robfallows — APPROVED** [review]
LGTM 👍 (other than the comment I added - feel free to ignore 🙂)

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (ben-moss) [reply .github/PULL_REQUEST_TEMPLATE.md:34]: Often they end up being the same, but really they are different things.

How you should test, versus the things that make this good enough as a solution

**Pull request merged**